src/Cognito.re

```diff
@@ -131,6 +132,7 @@ type signUpErrors = [
   | `CognitoUnexpectedLambda(string)
   | `CognitoUserLambdaValidation(string)
   | `CognitoUsernameExists(string)
+  | `CognitoConfirmationCodeValidation(string)
```

Comment:
Can you link me to where this error is documented?  I didn't see it when I was constructing this...

Reply:
That refers to https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_ConfirmSignUp.html. Second error in `Errors` section, "CodeMismatchException". Came up when I was seeing what happened if I put the wrong confirmation code for the code param.

Reply:
Ahh, yeah, let's organize that over in a type specific to confirm signup errors, and out of this type.  There will be some duplication but I don't think the type system cares given these are polymorphic variants.  If it does let me know and we can think up some solutions.